### PR TITLE
Update readme/documentations & fix parsing param directive

### PR DESCRIPTION
**Pull request by anru**
There is update for #17 PR which makes readme/documentation more consistent with latest codebase.

### CODE REVIEW

**Comment by anru** [comment]
It would be nice if somebody looked at it @mourner @IvanSanchez

File: src/leafdoc.js [hunk 1]
```diff
@@ -471,7 +471,7 @@ Leafdoc.prototype.addStr = function(str, isSource) {
 				} else if (directive === 'param') {
 					// Params are param name, type.
 					/// TODO: Think about default values, or param explanation.
-					var split = content.split(',');
+					var split = content.split(':');
```
Comment (IvanSanchez) [inline src/leafdoc.js:474]: Won't this change affect the parsing of functions with multiple parameters?

Comment (anru) [inline src/leafdoc.js:474]: This code works only for `@param foo: Type` constrcuctions, so it should not affect parsing of functions

Comment (IvanSanchez) [inline src/leafdoc.js:474]: You're right. It's been quite a long time since I split the `@param` directive and the shorthand syntax.

**Pull request merged**